### ISSUE TITLE
Data visualization vs Model Evaluation

### ISSUE THREAD

**Issue by choikwunyu** [EHWUSF/HS68_2018_Project_1#16]
People have been saying data visualization for all plotting related things, but I think we should split them between data visualization (more for understanding the dataset) and model evaluation (plots which help us assess the fit of the model to the data). What do you guys think?

**Comment by choikwunyu** [comment]
And of course some modules can have crossover, like @NikitaThomas's suggestion for labels/keys etc.

**Comment by RoxanneXin** [comment]
Yes, model evaluation will play an important role in linear regression. Like residual plots?
